libc/newhdrgen/header.py

```diff
@@ -0,0 +1,59 @@
+# class_implementation/classes/header.py
```

Comment:
Add license header and a bit more detailed description in the comments.

Reply:
just pushed comments and license header.